aws/wafv2_helper.go

```diff
@@ -862,20 +1069,107 @@ func flattenWafv2RuleAction(a *wafv2.RuleAction) interface{} {
 	m := map[string]interface{}{}
 
 	if a.Allow != nil {
-		m["allow"] = make([]map[string]interface{}, 1)
+		m["allow"] = flattenWafv2Allow(a.Allow)
 	}
 
 	if a.Block != nil {
-		m["block"] = make([]map[string]interface{}, 1)
+		m["block"] = flattenWafv2Block(a.Block)
 	}
 
 	if a.Count != nil {
-		m["count"] = make([]map[string]interface{}, 1)
+		m["count"] = flattenWafv2Count(a.Count)
 	}
 
 	return []interface{}{m}
 }
 
+func flattenWafv2Allow(a *wafv2.AllowAction) interface{} {
+	if a == nil {
+		return map[string]interface{}{}
+	}
+	m := map[string]interface{}{}
+
+	if a.CustomRequestHandling != nil {
+		m["custom_request_handling"] = flattenWafv2CustomRequestHandling(a.CustomRequestHandling)
+	}
+
+	return []interface{}{m}
+}
+
+func flattenWafv2Block(a *wafv2.BlockAction) interface{} {
+	if a == nil {
+		return map[string]interface{}{}
```

Comment:
similar comment as above
```suggestion
		return []interface{}{}
```